Source/Core/AudioCommon/AudioDevice.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+struct AudioDevice
+{
+
+public:
+	AudioDevice() {};
+	AudioDevice(u32 _index, std::string _name, std::string _path, std::string _id) : index(_index), name(_name), path(_path), id(_id) {};
```

Comment:
Don't use prefixed underscores in variable names, put the underscore after the name if necessary.